pkg/runtime/pubsub/subscriptions.go

```diff
@@ -214,7 +215,8 @@ func GetSubscriptionsGRPC(channel runtimev1pb.AppCallbackClient, log logger.Logg
 			if rErr != nil {
 				s, ok := status.FromError(rErr)
 				if ok && s != nil && s.Code() == codes.Unimplemented {
-					return rResp, backoff.Permanent(rErr)
+					log.Infof("pubsub subscriptions: gRPC app does not implement ListTopicSubscriptions")
```

Comment:
```suggestion
					log.Warnf("pubsub subscriptions: gRPC app does not implement ListTopicSubscriptions")
```

Reply:
I believe this should be Info since it is really not expected for apps to implement this.